imports/plugins/core/catalog/client/components/publishControls.js

```diff
@@ -13,7 +11,7 @@ import {
 import { Translatable } from "/imports/plugins/core/ui/client/providers";
 
 /** TMP **/
-import { i18next, Reaction } from "/client/api";
```

Comment:
I see other functions in here that are no longer used (1 related to i18n) I wonder if we shouldn't remove these are well?

Reply:
@zenweasel since this fixes the release-blocking bug and you've approved I'll merge. I'm all for removing unused code, so please go ahead with that, either PR or issue is 👍